particula/util/particle_mass.py

```diff
@@ -33,18 +39,13 @@ def mass(**kwargs):
         Returns:
                          (float) [kg]
     """
-
-    radius = kwargs.get("radius", None)
-    density = kwargs.get("density", 1000)
-    shape_factor = kwargs.get("shape_factor", 1)
-    vol_void = kwargs.get("volume_void", 0)
-
+    _ = kwargs.get("something", None)
```

Comment:
what is this? "something"?

Reply:
LOL, this was just to avoid upsetting the linter. The idea is that we should pass `kwargs` through, but we don't really need it here. If you don't use it, it will throw a linter error. This trick `_ = kwargs.get("something", None)` will use `kwargs` to read a keyword argument called "something", defaulting to `None`. Note that, effectively, `_` means it is just thrown out regardless and not used regardless; "something" here could be really anything. I can also just get rid of this and disable the linter warning/error instead.

Reply:
ha, I support the passing of kwargs everywhere :) as we build things out. so we're good here